Sources/AnimalKingdomAPI/graphql/WarmBloodedDetails.graphql

```diff
@@ -1,7 +1,4 @@
 fragment WarmBloodedDetails on WarmBlooded {
   bodyTemperature
-  height {
-    meters # TODO: Use HeightInMeters fragment?
-    yards
-  }
+  ...HeightInMeters  
```

Comment:
All of the changes to the generated code are because I did this. It's just a sanity check that merging works for fragments referenced from inside other fragments.